### ISSUE TITLE
ReactTable UI display issues

### ISSUE THREAD

**Issue by aaronjudd** [reactioncommerce/reaction#2557]
UI issues displaying empty tables.

![reaction](https://user-images.githubusercontent.com/439959/28221924-21cabaec-687a-11e7-934c-35a665a4bc63.png)

<img width="561" alt="reaction_and_1__rc__node__and_reactioncommerce_reaction" src="https://user-images.githubusercontent.com/439959/28221908-161e7c60-687a-11e7-9f6a-c890e955e202.png">

```
Node: 7.5.0
NPM: 5.0.0
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.10.2
Reaction: 1.3.1
Reaction branch: release-1.3.1
Docker: 17.03.1-ce
```

**Comment by kieckhafer** [comment]
The No Results UI is the default that comes with React-Table, we can definitely update it to look a little different, but I don't know if that's technically a bug so much an enhancement.

@rymorgan We briefly discussed this, but decided to stay with this just to get it pulled in. Any thoughts since then? 

The page dropdown on the second line probably shouldn't be happening.

**Comment by aaronjudd** [comment]
I think that this UI looks broken, and it's not an enhancement, but falls under UI QA, and is a bug...  we should clean this up.

**Comment by rymorgan** [comment]
@aaronjudd Worked with @kieckhafer on this yesterday. Here's the PR to check it out. We are aware there is an issue with the next button when pagination kicks in. @kieckhafer Is creating another issue for that. 

https://github.com/reactioncommerce/reaction/pull/2597

**Comment by kieckhafer** [comment]
Closed via #2597
